apps/web/components/booking/BookingListItem.tsx

```diff
@@ -396,6 +397,7 @@ function BookingListItem(booking: BookingItemProps) {
               {isRejected && <div className="text-sm text-gray-500">{t("rejected")}</div>}
             </>
           ) : null}
+          {isPast && isPending ? <>{!isConfirmed && <TableActions actions={bookedActions} />}</> : null}
```

Comment:
```suggestion
          {isPast && isPending && !isConfirmed ? <TableActions actions={bookedActions} /> : null}
```

Wouldn't this work as well?